### ISSUE TITLE
https requires refresh to show items other then the dashboard

### ISSUE THREAD

**Issue by staminna** [dreamerkumar/bizsitegenie#11]
Hi Vishal,

Using https, when I login, only the dashboard is shown. I always need to refresh so I can see the items created. Any idea?

I am adding [service workers support](https://github.com/GoogleChrome/samples/issues/462) though it is logging me out when I refresh and https is essencial for the app and the service workers to work. I think it is logging me out because I can't use document.querySelector() for angular authentication service. 
Thanks

**Comment by dreamerkumar** [comment]
I dont think it has got something to do with https. It has to be a caching issue. Its caching the previous state of thr home url. Thats  why refresh fixes the problem.

**Comment by staminna** [comment]
1. I disabled the service worker caches all together and still didn't show the main custom items at the first time. I have had this issue since I first started using htts, weeks before the service worker code.
2. I re-enabled the service worker code and now refresh doesn't log out anymore.

**Comment by dreamerkumar** [comment]
I was able to reproduce the issue. I don't know why it doesn't get the list of items after logging in the first time. But it works after refreshing the page. 

Will drill down further to figure out why this is so.

**Comment by staminna** [comment]
I've configured Google's Oauth mechanism, and it works flawlessly, no need to refresh.
In this particular case, the bootstrap is correct.
